src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -797,6 +797,21 @@ const schema = {
             type: 'string',
             enum: ['NONE', 'NEW', 'SECONDARY', 'INCREASE', 'REOPEN'],
           },
+          specialIssues: {
+            type: 'array',
+            items: {
+              type: 'object',
+              required: ['code', 'name'],
+              properties: {
+                name: {
+                  type: 'string',
+                },
+                code: {
+                  type: 'string',
```

Comment:
Per swagger, sounds like `code` should have `enums`? (Same for secondary.)

I think it's `['ALS', 'HEPC', 'POW', 'PTSD_1', 'PTSD_2', 'PTSD_4', 'PTSD_3']`

Reply:
@markgreenburg : enum added for the `code`